src/components/ProjectsDropdown/index.tsx

```diff
@@ -4,12 +4,17 @@ import { Dropdown, DropdownItem } from '@terraware/web-components';
 
 import { useLocalization } from 'src/providers';
 import strings from 'src/strings';
-import { Project } from 'src/types/Project';
+
+type AvailableProject = {
```

Comment:
Can rename this to ProjectDropdownOption to be consistent with our DropdownOption namings!